data/virt_autotest/host_unattended_installation_files/combustion/script

```diff
@@ -5,6 +5,8 @@ grub2_config() {
     file=/etc/default/grub
     sed -i 's/^\s*GRUB_TIMEOUT=.*$/GRUB_TIMEOUT=15/g' $file
     sed -i -r 's/^(\s|#)*GRUB_TERMINAL=.*$/GRUB_TERMINAL="gfxterm console"/g' $file
+    sed -i -r 's/^(\s|#)*GRUB_TERMINAL_INPUT=.*$/GRUB_TERMINAL_INPUT="gfxterm console"/g' $file
+    sed -i -r 's/^(\s|#)*GRUB_TERMINAL_OUTPUT=.*$/GRUB_TERMINAL_OUTPUT="gfxterm console"/g' $file
```

Comment:
Congratulation for masking a bug https://bugzilla.suse.com/show_bug.cgi?id=1230844